defaults/main.yml

```diff
@@ -102,6 +102,14 @@ jenkins_secret_files: []
 # can be useful to see which plugins changed during a deployment.
 jenkins_show_plugins_yaml_changes: true
 
+# How long to wait between retries when checking if Jenkins has started up. Using lower
+# values will reduce latency in the playbook at the expense of log spam.
+jenkins_startup_retry_delay_sec: 5
+
+# How long to wait for Jenkins to start before failing. Use a higher value if your Jenkins
+# instance has a lot of jobs and builds.
```

Comment:
I think we should mention that this should be greater than `jenkins_startup_retry_delay_sec`, because it does not make much sense to give a smaller value here.  What do you think?

Reply:
Let's assert this in `sanity-check.yml`. See f929449.